cmd/operator/deploy/crds/monitoring.googleapis.com_operatorconfigs.yaml

```diff
@@ -103,6 +103,30 @@ spec:
               object represents. Servers may infer this from the endpoint the client
               submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
+          managedAlertManager:
```

Comment:
`managed` is implied. I'd recommend `alertManager`. 

Also remember to update description etc and documentation in other places.

I think it should be at the same level as `rules` but weird that both are under `collection`. A better section would be `alerting`.. Ah well too late to change.

Reply:
Since there is already an `alertmanagers` field under `rules` for users to configure their own unmanaged AM, I think it is worth the distinction here that we are specifically talking about the "default"/managed alert manager

Also this is at the same level as `rules`, not under `collection`: https://github.com/GoogleCloudPlatform/prometheus-engine/blob/450382f6b3baa484941b0500c98b013830bd0d9c/pkg/operator/apis/monitoring/v1/types.go#L49-L58